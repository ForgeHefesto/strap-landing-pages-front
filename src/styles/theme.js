export const theme = {
  colors: {
    primaryColor: '#0A1128',
    secondaryColor: '#dc143C',
    lightBg: '#FFFFFF',
    darkBg: '#0A1128',
  },
  fonts: {
    family: {
      default: "'Open Sans', 'sans-serif;'",
      secondary: "'Montserrat', sans-serif;",
    },
    sizes: {
      xsmall: '8rem',
      small: '1.6rem',
      medium: '2.4rem',
      large: '3.2rem',
      xlarge: '4.0rem',
      xxlarge: '4.8rem',
      huge: '5.6rem',
      xhuge: '6.4rem',
    },
  },
  spacings: {
    xsmall: '0.8rem',
    small: '1.6rem',
    medium: '2.4rem',
    large: '3.2rem',
    xlarge: '4.0rem',
    xxlarge: '4.8rem',
    huge: '5.6rem',
    xhuge: '6.4rem',
  },
  media: {
    IteMedium: '(max-width: 768px)',
    IteLarge: '(max-width: 992px)',
    IteExtraLarge: '(max-width: 1200px)',
    IteExtraExtraLarge: '(max-width: 1400px)',
    IteExtraExtraExtraLarge: '(max-width: 1600px)',
    IteExtraExtraExtraExtraLarge: '(max-width: 1800px)',
  },
};
