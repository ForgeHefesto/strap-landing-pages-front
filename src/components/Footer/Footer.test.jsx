import { screen } from '@testing-library/react';
import { renderTheme } from '../../styles/render-theme';
import { Footer } from '.';

describe('<Footer />', () => {
  it('should render with default values', () => {
    const { container } = renderTheme(<Footer footerHtml={'<h1>Ola</aah1>'} />);
    expect(screen.getByRole('heading', { name: 'Ola' })).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
