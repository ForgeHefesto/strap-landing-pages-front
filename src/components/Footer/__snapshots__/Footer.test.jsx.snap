// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render with default values 1`] = `
.c0 {
  text-align: center;
}

.c0 a {
  color: inherit;
  text-decoration: none;
}

.c2 {
  font-size: 2.4rem;
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 0 3.2rem;
}

<div>
  <footer
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <h1>
          Ola
        </h1>
      </div>
    </div>
  </footer>
</div>
`;
